src/base/net/geoipdatabase.h

```diff
@@ -59,6 +59,8 @@ class GeoIPDatabase
 
 private:
     explicit GeoIPDatabase(quint32 size);
+    GeoIPDatabase(GeoIPDatabase&) = delete;
```

Comment:
I would prefer to add `Q_DISABLE_COPY_MOVE(GeoIPDatabase)` just above `Q_DECLARE_TR_FUNCTIONS(GeoIPDatabase)` instead.